### DISCUSSION TITLE
Can't find Wunda Smart on HACS

### DISCUSSION THREAD

**Discussion by Lletty** [tonyroberts/hawundasmart#19] [General]
Have added https://github.com/tonyroberts/hawundasmart as an Integration in HACS custom repository.  However, when we go to Settings/Devices and Services we do not see WundaSmart on the list.  When we try again, it tells us it already exists in store.  We have cleared cache and restarted HA several times.  Are we missing something?

**Comment by starnetz** [comment]
Hi @Lletty 
If you have not solved it by now - Search for hawundasmart

**Comment by tonyroberts** [comment]
You need to click the Add Integration button on the Devices & Services page after installing it via HACS
